cmd/server/assets/home.html

```diff
@@ -47,13 +47,13 @@ <h1>Issue Diagnosis Verification Code</h1>
       </div>
 
       <div class="row form-group">
-        <label for="testDate" class="col-sm-3">Test date</label>
+        <label for="symptomDate" class="col-sm-3">Symptom Onset Date</label>
         <div class="col-sm-9">
           <div class="datepicker date input-group p-0 shadow-sm">
-            <input type="date" id="testDate" name="testDate" min="{{.minDate}}" max="{{.maxDate}}" class="form-control" />
+            <input type="date" id="symptomDate" name="symptomDate" min="{{.minDate}}" max="{{.maxDate}}" class="form-control" />
           </div>
           <small class="form-text text-muted">
-            Optional, cannot be more than 14 days prior to today. This system is based on
+            <strong>Recommended</strong>: cannot be more than 14 days prior to today. This system is based on
```

Comment:
is this actually 'cannot'? Do we prevent a user from configuring it less?

Reply:
fixed - this now shows the configured value.